admin_guide/router.adoc

```diff
@@ -10,45 +10,47 @@
+Depending on the underlying implementation, you can monitor a running
+link:../architecture/core_objects/routing.html[router] in multiple ways. This
+topic discusses the HAProxy template router, and the components to check to
+ensure its health.
 
 == Viewing Statistics
-The HAProxy router exposes a web listener for the HAProxy statistics.  You can enter the public
-IP address of the router with the correctly configured port to view the statistics page.  You 
-will be prompted for the admin password.  This password and port were configured during the
-router installation.
+The HAProxy router exposes a web listener for the HAProxy statistics. Enter the
+router's public IP address and the correctly configured port (1936 by default)
+to view the statistics page and enter the admin password when prompted. This
+password and port were configured during the router installation, but can be
+found by viewing the *_haproxy.conf_* file on the container.
 
 == Viewing Logs
-To view a router log, simply run the `oc log` command for the pod.  Keep in mind that since
-the router is running as a plugin process that manages the underlying implementation the log
-that you will see is the log for the plugin, not the actual HAProxy log.  
+To view a router log, run the `oc log` command on the pod. Note that, since the
+router is running as a plug-in process that manages the underlying
+implementation, the log is for the plug-in, not the actual HAProxy log.
 
 == Router Internals
 .Routes.json
 
-As routes are processed by the HAProxy router they are stored both in memory, on disk, and in
-the HAProxy configuration file.  In `/var/lib/containers/router/routes.json` you can find the
-internal representation of the routes that is passed to the template to generate the HAProxy
-configuration file.  When troubleshooting a route issue it may help to look at this file to
-see the data that is being used to drive configuration.
+Routes are processed by the HAProxy router, so they are stored both in memory,
+on disk, and in the HAProxy configuration file. You can find the internal
+representation of the routes that is passed to the template to generate the
+HAProxy configuration file in the *_/var/lib/containers/router/routes.json_*
+file. When troubleshooting a routing issue, view this file to see which data is
+being used to drive configuration.
 
 .HAProxy Config
 
-In `/var/lib/haproxy/conf/haproxy.conf` you can find the configuration that is being used
-by HAProxy.  Here is where you can view the backends that have been created for specific routes.
-Also, in the same directory you can find the mapping files that are used by the helper 
-frontend and backends when mapping an incoming request to a backend.
+You can find the HAProxy configuration and the backends that have been created
+for specific routes in the *_/var/lib/haproxy/conf/haproxy.conf_* file. The
+mapping files used by the helper frontend and backends when mapping an incoming
```

Comment:
Hypenate front-end and back-end as per style guide. Also, this sentence feels awkward in construction to me. Just a nit, but I'd recommend something that starts with the end, like this:

This directory also contains mapping files that the helper front-end and back-ends use when mapping an incoming request.